package.json

```diff
@@ -4,6 +4,7 @@
   "version": "14.2.0",
   "dependencies": {
     "@unhead/vue": "^1.11.14",
+    "@vueuse/core": "^14.1.0",
```

Comment:
I added this just for longPress to tidy the scrap. Feels a little heavy handed for the singular use case but it should tree shake and it's a fairly popular and supposedly powerful lib so I'm interested to explore what else it might be handy for